python/tvm/relay/op/contrib/forge/forge_passes.py

```diff
@@ -147,13 +147,23 @@ def callback(self, pre, post, node_map):
         act = node_map[self.act][0]
         conv_pool = node_map[self.pattern][0]
         pad = node_map[self.pad][0]
+
         if not all(conv_pool.attrs.padding[i] == 0 for i in range(len(conv_pool.attrs.padding))) \
            or not isinstance(pad.args[1], tvm.relay.Constant) or not pad.args[1].data.shape == () \
            or not int(pad.args[1].data.numpy()) == 0:
            return
 
         pad_width = pad.attrs.pad_width
-        padding = list(pad_width[-2]) + list(pad_width[-3]) # left, right, top, bottom
+        top = pad_width[-2][0]
+        bottom = pad_width[-2][1]
+        left = pad_width[-1][0]
+        right = pad_width[-1][1]
+
+        assert top == bottom, "padding top and padding bottom must be same"
```

Comment:
Add brief more details about why these must be the same. Also, if this is a software limitation, let's add a link to the issue that will remove this constrain in the future.

Reply:
TTNN doesn't support different padding values. I will add the comments accordingly.